styles/nextjs/catppuccin.user.css

```
@@ -0,0 +1,174 @@
+  }
+
+  #catppuccin(@lookup, @accent) {
+    @rosewater: @catppuccin[@@lookup][@rosewater];
+    @flamingo: @catppuccin[@@lookup][@flamingo];
+    @pink: @catppuccin[@@lookup][@pink];
+    @mauve: @catppuccin[@@lookup][@mauve];
+    @red: @catppuccin[@@lookup][@red];
+    @maroon: @catppuccin[@@lookup][@maroon];
+    @peach: @catppuccin[@@lookup][@peach];
+    @yellow: @catppuccin[@@lookup][@yellow];
+    @green: @catppuccin[@@lookup][@green];
+    @teal: @catppuccin[@@lookup][@teal];
+    @sky: @catppuccin[@@lookup][@sky];
+    @sapphire: @catppuccin[@@lookup][@sapphire];
+    @blue: @catppuccin[@@lookup][@blue];
+    @lavender: @catppuccin[@@lookup][@lavender];
+    @text: @catppuccin[@@lookup][@text];
+    @subtext1: @catppuccin[@@lookup][@subtext1];
+    @subtext0: @catppuccin[@@lookup][@subtext0];
+    @overlay2: @catppuccin[@@lookup][@overlay2];
+    @overlay1: @catppuccin[@@lookup][@overlay1];
+    @overlay0: @catppuccin[@@lookup][@overlay0];
+    @surface2: @catppuccin[@@lookup][@surface2];
+    @surface1: @catppuccin[@@lookup][@surface1];
+    @surface0: @catppuccin[@@lookup][@surface0];
+    @base: @catppuccin[@@lookup][@base];
+    @mantle: @catppuccin[@@lookup][@mantle];
+    @crust: @catppuccin[@@lookup][@crust];
+    @accent-color: @catppuccin[@@lookup][@@accent];
+
+    --geist-foreground: @text;
+    --geist-background: @mantle;
+    --accents-1: @base;
+    --accents-2: @surface0;
+    --accents-3: @surface1;
+    --accents-4: @surface2;
+    --accents-5: @overlay0;
+    --accents-6: @overlay1;
+    --accents-7: @overlay2;
+    --accents-8: @subtext0;
+    --geist-link-color: @rosewater;
+    --geist-selection: @subtext0;
+    --geist-success: @accent-color;
+    --geist-error: @red;
+    --geist-cyan: @teal;
+
+    --ds-background-100: @base;
+    --ds-background-200: @surface0;
+    --ds-gray-100: @base;
+    --ds-gray-200: @surface0;
+    --ds-gray-300: @surface1;
+    --ds-gray-1000: @text;
+    --ds-gray-900: @subtext0;
+    --ds-gray-800: @subtext1;
+
+    --ds-red-800: @red;
+    --ds-red-900: lighten(@red, 5%);
+    
+    /* Links vertical line */
+    .bg-blue-600 {
+      background-color: @accent-color;
+    }
+
+    /* App Router icon */
+    --ds-blue-700: @blue;
+
+    /* Pages Router icon */
+    --ds-purple-700: @mauve;
+
+    --ds-blue-400: {
+      color: @base;
+    };
+
+    --ds-purple-400: {
+      color: @base;
+    };
+
+    --tw-gradient-top: transparent;
+
+    a h3 {
+      color: @accent-color;
+    }
+
+    p a {
+      color: @accent-color;
+    }
+
+    .text-blue-700 {
+      color: @accent-color;
+    }
+
+    svg {
+      --geist-stroke: var(--geist-background) !important;
+    }
+
+    button.navbar_search__dZT2b {
+      background-color: @mantle !important;
+    }
+
+    input::placeholder {
+      color: @text;
+    }
```

Also themes the fake placeholder text in the search bar at the top of the page.
```suggestion
    input::placeholder,
    button[class*="navbar_search"] {
      color: @text;
    }
```